torch/serialization.py

```diff
@@ -1470,6 +1470,11 @@ def _get_wo_message(message: str) -> str:
         if pickle_module is None:
             pickle_module = pickle
 
+    if pickle_load_args is not None and weights_only == True:
```

Comment:
Looking at the tests that failed, the issue doesn't seem to be that `pickle_load_args` was passed, but rather that the use of `None` was wrong here

```suggestion
    if pickle_load_args != {} and weights_only == True:
```